features/step_definitions/branch_steps.rb

```diff
@@ -33,7 +33,7 @@
 end
 
 
-Given(/^I have a feature branch named "([^"]+)" as a child of "([^"]+)"$/) do |branch_name, parent_name|
+Given(/^I have a(?: feature)?(?: hotfix)? branch named "([^"]+)" as a child of "([^"]+)"$/) do |branch_name, parent_name|
```

Comment:
can this be `(?: feature| hotfix)`?. I think thats better as we don't ever want both.